src/internet.ts

```diff
@@ -426,4 +438,25 @@ export class Internet {
     };
     return _password(len, memorable, pattern, prefix);
   }
+
+  /**
+   * Generates a random emoji.
+   *
+   * @param options Options object.
+   * @param options.types A list of the emoji types that should be used.
+   * @example
+   * faker.internet.emoji() // '🥰'
+   * faker.internet.emoji({types: ['food', 'nature']}) // '🥐'
```

Comment:
@ST-DDT will api-docs ignore such unformatted example docs? Or should we care about that this should be formatted?
```suggestion
   * faker.internet.emoji({ types: ['food', 'nature'] }) // '🥐'
```